src/cpp/rtps/builtin/discovery/participant/PDPSimple.cpp

```diff
@@ -737,11 +737,6 @@ void PDPSimple::assign_low_level_remote_endpoints(
         const ParticipantProxyData& pdata,
         bool notify_secure_endpoints)
 {
-    if (mp_EDP != nullptr)
-    {
-        mp_EDP->assignRemoteEndpoints(pdata, notify_secure_endpoints);
-    }
-
```

Comment:
We might need a similar change in `PDPServer::perform_builtin_endpoints_matching`